apis/Google.Cloud.Bigtable.V2/Google.Cloud.Bigtable.V2.Tests/MutateRowsStreamUtilities.cs

```diff
@@ -46,7 +46,7 @@ public static class MutateRowsStreamUtilities
             mock.Setup(c => c.MutateRows(initialRequest, It.IsAny<CallSettings>()))
                 .Returns(StreamFromEntries(entriesForInitialResponse));
 
-            return new BigtableClientImpl(mock.Object, appProfileId: null);
+            return new BigtableClientImpl(null, mock.Object);
```

Comment:
If the params in the constructor was added just so we could do this, I would prefer not to have it. I would rather the self-documentation of the named argument if that means creating an array explicitly:
```csharp
return new BigtableClientImpl(new [] { mock.Object }, appProfileId: null);
```